### ISSUE TITLE
the size of Room B

### ISSUE THREAD

**Issue by bingo-todd** [IoSR-Surrey/RealRoomBRIRs#1]
The width and length of room B are the same, which is not consist with the rectangular shape of this room.

**Comment by TuttiChi** [comment]
You are right.And the length obviously wrong because the listener position add 1.5m is beyond the length.
